app/views/admin/kaminari/_first_page.html.erb

```diff
@@ -0,0 +1,3 @@
+<li>
+  <%= link_to_unless current_page.first?, raw(t 'views.pagination.first'), url, :remote => remote %>
```

Comment:
These were imported by kaminari.  I believe it's a rails-ujs thing I can take out.